projects/coordinator/src/test/java/org/batfish/coordinator/WorkMgrServiceV2Test.java

```diff
@@ -54,32 +60,48 @@ protected Application configure() {
   }
 
   @Test
-  public void getContainers() throws Exception {
-    Response response = target("/v2/containers").request().get();
+  public void getContainers() {
+    Response response = target(BASE_CONTAINER_PATH).request().get();
```

Comment:
target(V2_CONSTANT).path(CONTAINERS)
target(V2_CONSTANT).path(CONTAINER)
target(V2_CONSTANT).path(CONTAINER).path(ID)

(something like that)

and then you don't need to use Paths to join URLs -- that's what chained calls to target are for.

Reply:
or you could make a private static function getContainers() that returns a Response and contains this whole line (with proper targeting)

and another one getContainer(String id)
...